src/main/java/com/ericsson/ei/controller/DownloadControllerImpl.java

```diff
@@ -57,7 +59,7 @@
             return new ResponseEntity<>(IOUtils.toByteArray(is), HttpStatus.OK);
         } catch (IOException e) {
             LOGGER.error(e.getMessage(), e);
-            return new ResponseEntity<>("File is not found", HttpStatus.NOT_FOUND);
+            return new ResponseEntity<>(NOT_FOUND, HttpStatus.NOT_FOUND);
```

Comment:
Can you add info of what file that couldn't be found. Same comment for the other Responses.